### PR TITLE
Update demo pipeline dependencies to support Gradle 7

### PR DESCRIPTION
**Pull request by disrupted**

### CODE REVIEW

**Comment by philipp94831** [comment]
Why don't you also update Gradle in this PR?

**Comment by disrupted** [comment]
> Why don't you also update Gradle in this PR?

wasn't aware that the gradle wrapper has its own version

File: demo-atm-fraud/gradlew.bat [hunk 1]
```diff
@@ -64,29 +64,14 @@ echo location of your Java installation.
 
 goto fail
 
-:init
```
Comment (philipp94831) [inline demo-atm-fraud/gradlew.bat:67]: I assume this was automatically done?

Comment (disrupted) [reply demo-atm-fraud/gradlew.bat:67]: yes

**Review by philipp94831 — APPROVED** [review]

**Pull request merged**